test/integration/sagemaker/test_mnist.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+from __future__ import absolute_import
+
+import os
+
+from sagemaker.tensorflow import TensorFlow
+
+
+def test_mnist(sagemaker_session, ecr_image, instance_type):
+    _run_minist_training(sagemaker_session, ecr_image, instance_type)
+
+
+def _run_minist_training(sagemaker_session, ecr_image, instance_type):
```

Comment:
s/minist/mnist

Reply:
I thought about just move this to the test but kept it because I plan to add distributed training tests later. And tests for creating endpoint and inference will be added too.

Reply:
Actually this function probably will need refactoring anyway. i will merge this in the test for now.